app/models/user.rb

```diff
@@ -85,6 +85,10 @@ def name
     "#{firstname} #{lastname}"
   end
 
+  def unavailable
```

Comment:
```suggestion
  def unavailable?
```
Methods returning a boolean normally end with a question mark :)